From 17850193d6a3a19f3296e286e4cc045fab60e7cb Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Wed, 27 Feb 2019 00:11:55 +0200
Subject: [PATCH 2/4] Bluetooth: Remove unused string resources

strings_map.xml was removed in 9905d4a931fb1841afb4a3252954af5751691668
but it was mistakenly added back for some languages in
b9b4c8a5d791800f7249f80cd5de0e150ed033f4

Change-Id: Id49aab0526cc571fba60f2e80f56e64b3f1cd9f5
---
 res/values-az/strings_map.xml | 15 ---------------
 res/values-et/strings_map.xml | 15 ---------------
 res/values-hy/strings_map.xml | 15 ---------------
 res/values-ka/strings_map.xml | 15 ---------------
 res/values-km/strings_map.xml | 15 ---------------
 res/values-lo/strings_map.xml | 15 ---------------
 res/values-mn/strings_map.xml | 15 ---------------
 res/values-ms/strings_map.xml | 15 ---------------
 res/values-ne/strings_map.xml | 15 ---------------
 res/values-si/strings_map.xml | 15 ---------------
 10 files changed, 150 deletions(-)
 delete mode 100644 res/values-az/strings_map.xml
 delete mode 100644 res/values-et/strings_map.xml
 delete mode 100644 res/values-hy/strings_map.xml
 delete mode 100644 res/values-ka/strings_map.xml
 delete mode 100644 res/values-km/strings_map.xml
 delete mode 100644 res/values-lo/strings_map.xml
 delete mode 100644 res/values-mn/strings_map.xml
 delete mode 100644 res/values-ms/strings_map.xml
 delete mode 100644 res/values-ne/strings_map.xml
 delete mode 100644 res/values-si/strings_map.xml

diff --git a/res/values-az/strings_map.xml b/res/values-az/strings_map.xml
deleted file mode 100644
index 0b8584e1..00000000
--- a/res/values-az/strings_map.xml
+++ /dev/null
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<resources xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="map_session_key_dialog_title" msgid="4357431314165953502">"%1$s üçün seans açarı daxil edin"</string>
-    <string name="map_session_key_dialog_header" msgid="1858363785687455516">"Bluetooth seans açarı tələb olunur"</string>
-    <string name="map_acceptance_timeout_message" msgid="2247454902690565876">"%1$s ilə bağlantı qəbul edərkən fasilə yarandı"</string>
-    <string name="map_authentication_timeout_message" msgid="2151423397615327066">"%1$s ilə seans açarını daxil edərkən fasilə yarandı"</string>
-    <string name="map_auth_notif_ticker" msgid="8840174807927327119">"Obex təsdiqləmə sorğusu"</string>
-    <string name="map_auth_notif_title" msgid="301767019364765129">"Seans Açarı"</string>
-    <string name="map_auth_notif_message" msgid="1510095655064214863">"%1$s üçün seans açarı daxil edin"</string>
-    <string name="map_defaultname" msgid="3805234816465216759">"Maşın dəsti"</string>
-    <string name="map_unknownName" msgid="4985352365863687360">"Naməlum ad"</string>
-    <string name="map_localPhoneName" msgid="6264496105941797768">"Mənim adım"</string>
-    <string name="map_defaultnumber" msgid="3838264444043503815">"000000"</string>
-</resources>
diff --git a/res/values-et/strings_map.xml b/res/values-et/strings_map.xml
deleted file mode 100644
index 8fe16424..00000000
--- a/res/values-et/strings_map.xml
+++ /dev/null
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<resources xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="map_session_key_dialog_title" msgid="4357431314165953502">"Sisestage seansivõti seadmele %1$s"</string>
-    <string name="map_session_key_dialog_header" msgid="1858363785687455516">"Vajalik Bluetoothi seansivõti"</string>
-    <string name="map_acceptance_timeout_message" msgid="2247454902690565876">"Esines ajalõpp ühenduse aktsepteerimiseks seadmega %1$s"</string>
-    <string name="map_authentication_timeout_message" msgid="2151423397615327066">"Esines ajalõpp seansivõtme sisestamiseks seadmele %1$s"</string>
-    <string name="map_auth_notif_ticker" msgid="8840174807927327119">"Obexi autentimise taotlus"</string>
-    <string name="map_auth_notif_title" msgid="301767019364765129">"Seansivõti"</string>
-    <string name="map_auth_notif_message" msgid="1510095655064214863">"Sisestage seansivõti seadmele %1$s"</string>
-    <string name="map_defaultname" msgid="3805234816465216759">"Autokomplekt"</string>
-    <string name="map_unknownName" msgid="4985352365863687360">"Tundmatu nimi"</string>
-    <string name="map_localPhoneName" msgid="6264496105941797768">"Minu nimi"</string>
-    <string name="map_defaultnumber" msgid="3838264444043503815">"000000"</string>
-</resources>
diff --git a/res/values-hy/strings_map.xml b/res/values-hy/strings_map.xml
deleted file mode 100644
index 91f2078b..00000000
--- a/res/values-hy/strings_map.xml
+++ /dev/null
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<resources xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="map_session_key_dialog_title" msgid="4357431314165953502">"Մուտքագրեք աշխատաշրջանի բանալին %1$s-ի համար"</string>
-    <string name="map_session_key_dialog_header" msgid="1858363785687455516">"Պահանջվում է Bluetooth-ի աշխատաշրջանի բանալի"</string>
-    <string name="map_acceptance_timeout_message" msgid="2247454902690565876">"%1$s-ի հետ կապի ընդունման ժամանակը սպառվեց"</string>
-    <string name="map_authentication_timeout_message" msgid="2151423397615327066">"%1$s-ով աշխատաշրջանի բանալու մուտքագրման ժամանակը սպառվեց"</string>
-    <string name="map_auth_notif_ticker" msgid="8840174807927327119">"Կասեցնել նույնականացման հարցումը"</string>
-    <string name="map_auth_notif_title" msgid="301767019364765129">"Աշխատաշրջանի բանալի"</string>
-    <string name="map_auth_notif_message" msgid="1510095655064214863">"Մուտքագրեք աշխատաշրջանի բանալին %1$s-ի համար"</string>
-    <string name="map_defaultname" msgid="3805234816465216759">"Carkit"</string>
-    <string name="map_unknownName" msgid="4985352365863687360">"Անհայտ անուն"</string>
-    <string name="map_localPhoneName" msgid="6264496105941797768">"Իմ անունը"</string>
-    <string name="map_defaultnumber" msgid="3838264444043503815">"000000"</string>
-</resources>
diff --git a/res/values-ka/strings_map.xml b/res/values-ka/strings_map.xml
deleted file mode 100644
index 8e645c4f..00000000
--- a/res/values-ka/strings_map.xml
+++ /dev/null
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<resources xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="map_session_key_dialog_title" msgid="4357431314165953502">"სესიის გასაღები %1$s-ისთვის"</string>
-    <string name="map_session_key_dialog_header" msgid="1858363785687455516">"აუცილებელია Bluetooth სესიის გასაღები"</string>
-    <string name="map_acceptance_timeout_message" msgid="2247454902690565876">"%1$s-თან კავშირის მიღების დრო ამოიწურა"</string>
-    <string name="map_authentication_timeout_message" msgid="2151423397615327066">"%1$s-თან სესიის გასაღების შეყვანის დრო ამოიწურა"</string>
-    <string name="map_auth_notif_ticker" msgid="8840174807927327119">"Obex ავთენტიფიკაციის მოთხოვნა"</string>
-    <string name="map_auth_notif_title" msgid="301767019364765129">"სესიის გასაღები"</string>
-    <string name="map_auth_notif_message" msgid="1510095655064214863">"სესიის გასაღები %1$s-ისთვის"</string>
-    <string name="map_defaultname" msgid="3805234816465216759">"მანქანის ნაკრები"</string>
-    <string name="map_unknownName" msgid="4985352365863687360">"უცნობი სახელი"</string>
-    <string name="map_localPhoneName" msgid="6264496105941797768">"ჩემი სახელი"</string>
-    <string name="map_defaultnumber" msgid="3838264444043503815">"000000"</string>
-</resources>
diff --git a/res/values-km/strings_map.xml b/res/values-km/strings_map.xml
deleted file mode 100644
index 81186698..00000000
--- a/res/values-km/strings_map.xml
+++ /dev/null
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<resources xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="map_session_key_dialog_title" msgid="4357431314165953502">"បញ្ចូល​សោ​សម័យ​សម្រាប់ %1$s"</string>
-    <string name="map_session_key_dialog_header" msgid="1858363785687455516">"បាន​ទាមទារ​សោ​សម័យ​ប៊្លូធូស"</string>
-    <string name="map_acceptance_timeout_message" msgid="2247454902690565876">"អស់​ពេល​វេលា​ក្នុង​ការ​ទទួល​យក​ការ​តភ្ជាប់​ជាមួយ %1$s"</string>
-    <string name="map_authentication_timeout_message" msgid="2151423397615327066">"អស់​ពេល​​ដើម្បី​បញ្ចូល​សោ​សម័យ​ជាមួយ %1$s"</string>
-    <string name="map_auth_notif_ticker" msgid="8840174807927327119">"សំណើ​ការ​ផ្ទៀងផ្ទាត់ Obex"</string>
-    <string name="map_auth_notif_title" msgid="301767019364765129">"សោ​សម័យ"</string>
-    <string name="map_auth_notif_message" msgid="1510095655064214863">"បញ្ចូល​សោ​សម័យ​សម្រាប់ %1$s"</string>
-    <string name="map_defaultname" msgid="3805234816465216759">"Carkit"</string>
-    <string name="map_unknownName" msgid="4985352365863687360">"មិន​ស្គាល់​ឈ្មោះ"</string>
-    <string name="map_localPhoneName" msgid="6264496105941797768">"ឈ្មោះ​របស់​ខ្ញុំ"</string>
-    <string name="map_defaultnumber" msgid="3838264444043503815">"000000"</string>
-</resources>
diff --git a/res/values-lo/strings_map.xml b/res/values-lo/strings_map.xml
deleted file mode 100644
index dd3a2889..00000000
--- a/res/values-lo/strings_map.xml
+++ /dev/null
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<resources xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="map_session_key_dialog_title" msgid="4357431314165953502">"ພິມລະຫັດເຊສຊັນສຳລັບ %1$s"</string>
-    <string name="map_session_key_dialog_header" msgid="1858363785687455516">"ຕ້ອງມີ Bluetooth ລະຫັດເຊສຊັນ"</string>
-    <string name="map_acceptance_timeout_message" msgid="2247454902690565876">"ໝົດເວລາທີ່ຈະຮັບການເຊື່ອມຕໍ່ກັບ %1$s"</string>
-    <string name="map_authentication_timeout_message" msgid="2151423397615327066">"ເກີດໝົດເວລາກ່ອນທີ່ຈະໃສ່ລະຫັດເຊສຊັນກັບ %1$s"</string>
-    <string name="map_auth_notif_ticker" msgid="8840174807927327119">"ການຮ້ອງຂໍພິສູດຢືນຢັນໂຕ Obex"</string>
-    <string name="map_auth_notif_title" msgid="301767019364765129">"ກະແຈເຊສຊັສ"</string>
-    <string name="map_auth_notif_message" msgid="1510095655064214863">"ພິມລະຫັດເຊສຊັນສຳລັບ %1$s"</string>
-    <string name="map_defaultname" msgid="3805234816465216759">"Carkit"</string>
-    <string name="map_unknownName" msgid="4985352365863687360">"ບໍ່ຮູ້ຊື່"</string>
-    <string name="map_localPhoneName" msgid="6264496105941797768">"ຊື່ຂອງຂ້ອຍ"</string>
-    <string name="map_defaultnumber" msgid="3838264444043503815">"000000"</string>
-</resources>
diff --git a/res/values-mn/strings_map.xml b/res/values-mn/strings_map.xml
deleted file mode 100644
index 45fc07dd..00000000
--- a/res/values-mn/strings_map.xml
+++ /dev/null
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<resources xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="map_session_key_dialog_title" msgid="4357431314165953502">"%1$s-н горимын түлхүүрийг оруулна уу"</string>
-    <string name="map_session_key_dialog_header" msgid="1858363785687455516">"Блютүүт горимын түлхүүр шаардлагатай"</string>
-    <string name="map_acceptance_timeout_message" msgid="2247454902690565876">"%1$s-тай холболт хийхийг зөвшөөрөх явцад хугацаа хэтэрсэн байна"</string>
-    <string name="map_authentication_timeout_message" msgid="2151423397615327066">"%1$s-д горимын түлхүүр оруулах явцад хугацаа хэтэрсэн байна"</string>
-    <string name="map_auth_notif_ticker" msgid="8840174807927327119">"Obex гэрчлэлтийн хүсэлт"</string>
-    <string name="map_auth_notif_title" msgid="301767019364765129">"Горимын Түлхүүр"</string>
-    <string name="map_auth_notif_message" msgid="1510095655064214863">"%1$s-н горимын түлхүүрийг оруулна уу"</string>
-    <string name="map_defaultname" msgid="3805234816465216759">"Carkit"</string>
-    <string name="map_unknownName" msgid="4985352365863687360">"Тодорхойгүй нэр"</string>
-    <string name="map_localPhoneName" msgid="6264496105941797768">"Миний нэр"</string>
-    <string name="map_defaultnumber" msgid="3838264444043503815">"000000"</string>
-</resources>
diff --git a/res/values-ms/strings_map.xml b/res/values-ms/strings_map.xml
deleted file mode 100644
index afc98d73..00000000
--- a/res/values-ms/strings_map.xml
+++ /dev/null
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<resources xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="map_session_key_dialog_title" msgid="4357431314165953502">"Taipkan kunci sesi untuk %1$s"</string>
-    <string name="map_session_key_dialog_header" msgid="1858363785687455516">"Kunci sesi Bluetooth diperlukan"</string>
-    <string name="map_acceptance_timeout_message" msgid="2247454902690565876">"Berlaku tamat masa semasa menerima sambungan dengan %1$s"</string>
-    <string name="map_authentication_timeout_message" msgid="2151423397615327066">"Tamat masa berlaku semasa memasukkan kunci sesi dengan %1$s"</string>
-    <string name="map_auth_notif_ticker" msgid="8840174807927327119">"Permintaan pengesahan Obex"</string>
-    <string name="map_auth_notif_title" msgid="301767019364765129">"Kunci Sesi"</string>
-    <string name="map_auth_notif_message" msgid="1510095655064214863">"Taipkan kunci sesi untuk %1$s"</string>
-    <string name="map_defaultname" msgid="3805234816465216759">"Kit kereta"</string>
-    <string name="map_unknownName" msgid="4985352365863687360">"Nama tidak diketahui"</string>
-    <string name="map_localPhoneName" msgid="6264496105941797768">"Nama saya"</string>
-    <string name="map_defaultnumber" msgid="3838264444043503815">"000000"</string>
-</resources>
diff --git a/res/values-ne/strings_map.xml b/res/values-ne/strings_map.xml
deleted file mode 100644
index 7584c1ec..00000000
--- a/res/values-ne/strings_map.xml
+++ /dev/null
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<resources xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="map_session_key_dialog_title" msgid="4357431314165953502">"%1$s का लागि सत्र कुञ्जी टाइप गर्नुहोस्"</string>
-    <string name="map_session_key_dialog_header" msgid="1858363785687455516">"ब्लूटूथ सत्र कुञ्जी आवश्यक"</string>
-    <string name="map_acceptance_timeout_message" msgid="2247454902690565876">"%1$s सँग जडान स्वीकार गर्न समय सकियो"</string>
-    <string name="map_authentication_timeout_message" msgid="2151423397615327066">"%1$s का साथ सत्र कुञ्जी इनपुट गर्ने समय सकियो"</string>
-    <string name="map_auth_notif_ticker" msgid="8840174807927327119">"Obex प्रमाणिकरण अनुरोध"</string>
-    <string name="map_auth_notif_title" msgid="301767019364765129">"सत्र कुञ्जी"</string>
-    <string name="map_auth_notif_message" msgid="1510095655064214863">"%1$s का लागि सत्र कुञ्जी टाइप गर्नुहोस्"</string>
-    <string name="map_defaultname" msgid="3805234816465216759">"Carkit"</string>
-    <string name="map_unknownName" msgid="4985352365863687360">"अज्ञात नाम"</string>
-    <string name="map_localPhoneName" msgid="6264496105941797768">"मेरो नाम"</string>
-    <string name="map_defaultnumber" msgid="3838264444043503815">"००००००"</string>
-</resources>
diff --git a/res/values-si/strings_map.xml b/res/values-si/strings_map.xml
deleted file mode 100644
index 31c2febc..00000000
--- a/res/values-si/strings_map.xml
+++ /dev/null
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<resources xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="map_session_key_dialog_title" msgid="4357431314165953502">"%1$s සඳහා සැසි යතුර ටයිප් කරන්න"</string>
-    <string name="map_session_key_dialog_header" msgid="1858363785687455516">"බ්ලූටූත් සැසි යතුර අවශ්‍යයි"</string>
-    <string name="map_acceptance_timeout_message" msgid="2247454902690565876">"%1$s හා සම්බන්ධය පිළිගැනීමට කාල නිමාවක්  විය"</string>
-    <string name="map_authentication_timeout_message" msgid="2151423397615327066">"%1$s හා සැසි යතුර ආදානය කිරීමට කාල නිමාවක් විය"</string>
-    <string name="map_auth_notif_ticker" msgid="8840174807927327119">"Obex සත්‍යාපන ඉල්ලීම"</string>
-    <string name="map_auth_notif_title" msgid="301767019364765129">"සැසි යතුර"</string>
-    <string name="map_auth_notif_message" msgid="1510095655064214863">"%1$s සඳහා සැසි යතුර ටයිප් කරන්න"</string>
-    <string name="map_defaultname" msgid="3805234816465216759">"Carkit"</string>
-    <string name="map_unknownName" msgid="4985352365863687360">"නොදන්නා නමකි"</string>
-    <string name="map_localPhoneName" msgid="6264496105941797768">"මගේ නම"</string>
-    <string name="map_defaultnumber" msgid="3838264444043503815">"000000"</string>
-</resources>
-- 
2.17.1

